float64 dt
int8 steps
---
Obstacles result
